content/en/news/2019/announcing-1.4/change-notes/index.md

```diff
@@ -16,8 +16,13 @@ weight: 10
 
 ## Telemetry
 
+- **Added** experimental in-proxy telemetry reporting to [Stackdriver](https://github.com/istio/proxy/blob/release-1.4/extensions/stackdriver/README.md).
+- **Added** experimental support for generating TCP service metrics [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/).
+- **Improved** support for [in-proxy](/docs/ops/telemetry/in-proxy-service-telemetry/) Prometheus generation of HTTP service metrics (from experimental to alpha).
+- **Improved** telemetry collection for [blocked and passthrough external service traffic](/blog/2019/monitoring-external-service-traffic/).
```

Comment:
Is there a doc rather than a blog post this can point to?

Reply:
not that I know of @nrjpoddar?

Reply:
We don't have a doc for blocked and passthrough, I only added the blog which is linked few weeks ago.

Reply:
In general, we don't want the blog to used as the source of truth for product usage info. So if you have some time @nrjpoddar, please add a doc page for this.

Reply:
Should we merge this and update if that doc does exist?

Reply:
Yes, we should proceed as-is.

Reply:
> In general, we don't want the blog to used as the source of truth for product usage info. So if you have some time @nrjpoddar, please add a doc page for this.

Yep, I can do that after KubeCon. P&T tasks in our docs is very slim anyways so adding stuff there would be good.